Reading List/Reading List/Views/BookTableViewCell.swift

```diff
@@ -0,0 +1,52 @@
+//
+//  BookTableViewCell.swift
+//  Reading List
+//
+//  Created by Angel Buenrostro on 1/18/19.
+//  Copyright © 2019 Lambda School. All rights reserved.
+//
+
+import UIKit
+
+class BookTableViewCell: UITableViewCell, BookTableViewCellDelegate {
+    
+    var book: Book?
```

Comment:
You have everything but this:
```suggestion
        var book: Book? {
        didSet { updateViews() }
    }
```
You need to call `updateViews` function whenever `book`'s value changed. Otherwise cell's label text isn't changing